tests/pagination.tests.ts

```diff
@@ -148,4 +148,57 @@ describe('Pagination browser test', () => {
     const hits = response.results[0].hits
     expect(hits.length).toBe(6)
   })
+
+  test('Test bigger pagination total hits than nbr hits', async () => {
+    const customClient = instantMeiliSearch(
+      'http://localhost:7700',
+      'masterKey',
+      {
+        paginationTotalHits: 1000,
+      }
+    )
+    const response = await customClient.search<Movies>([
+      {
+        indexName: 'movies',
+      },
+    ])
+    const hits = response.results[0].hits
+    expect(hits.length).toBe(6)
+  })
+
+  test('Test pagination total hits with finite pagination', async () => {
+    const customClient = instantMeiliSearch(
+      'http://localhost:7700',
+      'masterKey',
+      {
+        paginationTotalHits: 5,
+        finitePagination: true,
+      }
+    )
+    const response = await customClient.search<Movies>([
+      {
+        indexName: 'movies',
+      },
+    ])
+    const hits = response.results[0].hits
```

Comment:
Can you add a line breaking to separate the test scopes? 

```js
//setup
const customClient = instantMeiliSearch(...)

// exercise sut
const response = await customClient.search<Movies>(...)

// assert
const hits = ...
expect(...)
```

Reply:
See https://github.com/meilisearch/instant-meilisearch/pull/727